src/Nest/Cluster/NodesUsage/NodeUsageResponse.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public class NodeUsageMetadata
```

Comment:
Actually, I think the name should be `NodesMetaData`, which mirrors the existing `ShardsMetaData`

Reply:
:+1:

Reply:
I think `MetaData` suffix is a mistake in my part. We have both `Metadata` and `MetaData` suffixes in the code base and I've been leaning towards the first since that mimmics elasticsearch's snake casing of `metadata`.  Maybe in 6.0 we do a sweep to standardize, worth the breaking change?